core/src/main/resources/error/error-classes.json

```diff
@@ -256,6 +256,18 @@
       "Key <keyValue> does not exist. Use `try_element_at` to tolerate non-existent key and return NULL instead. If necessary set <config> to \"false\" to bypass this error."
     ]
   },
+  "MELT_REQUIRES_VALUE_COLUMNS" : {
+    "message" : [
+      "At least one non-id column is required to melt. All columns are id columns: [<ids>]"
```

Comment:
I don't understand this restriction. What does it mean?

Reply:
Maybe clearer to say: `At least one value column needs to be specified for UNPIVOT.`

Reply:
done